EmployeeDepartment/src/main/java/com/example/EmployeeDepartment/controller/DepartmentController.java

```diff
@@ -0,0 +1,47 @@
+package com.example.EmployeeDepartment.controller;
+
+import com.example.EmployeeDepartment.entity.Department;
+import com.example.EmployeeDepartment.entity.Employee;
+import com.example.EmployeeDepartment.services.DepartmentService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.List;
+
+@Controller
+@RequestMapping("/departments")
+public class DepartmentController {
+    @Autowired
+    private DepartmentService departmentService;
+    public static int departmentid;
```

Comment:
Why are you using this variable? If its to store user id between API calls. Its wrong